apps/ledger-live-desktop/src/renderer/families/elrond/components/Modals/Claim/fields/DelegationSelectorField.tsx

```diff
@@ -45,7 +45,7 @@ const renderItem = (item: { data: OptionType }) => {
       </Box>
 
       <Text ff="Inter|Regular">
-        {balance} {"EGLD"} {/* FIXME Should be getAccountUnit(account).code */}
```

Comment:
We are not fixing this one because it's not easy to get the account in this `renderItem` or just no need for a fix ?

Reply:
`getAccountUnit(account)` as the comment is refering too is dropped by this PR, so i just removed the comment.

This is what we did overall on this PR: we stay on current status quo but we schedule with product possible improvements (for instance, as explained in description, we will have opportunity to use the unit at the currency level, so for instance in asset distribution lists-

Indeed also there are possible improvements on coin implementations: i noticed a bunch of inconsistencies on coin/account custom components that don't use the account unit at all but it wasn't part of the scope to fix these here (e.g. on Cosmos custom ui)